scripts/api_review/README.md

```diff
@@ -0,0 +1,2 @@
+# Elektra API Review Template Script
+This python script generates new review files from the given template.
```

Comment:
You can put this line directly as comment at the top of scripts/api_review/generate_review_files.py